### PR TITLE
Document MyST syntax

### PR DESCRIPTION
**Pull request by ssbarnea**
Fixes: #43

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -51,6 +51,13 @@ choose an alternate working directory with the ``cwd`` option.  The argument of
 this option is either a path relative to the current source file, or a absolute
 path which means that it is relative to the top level source directory.
 
+Usage with MyST (Markdown)
+==========================
+
+If you are using MyST_, the directive would look like::
+
+    ```{program-output} python -V
```
Comment (jamadden) [inline doc/index.rst:59]: How would one specify options, like `:ellipsis:` in this syntax?

Comment (ssbarnea) [reply doc/index.rst:59]: No elipses, three backquotes are needed as documented on the link: https://myst-parser.readthedocs.io/en/latest/using/syntax.html#extended-block-tokens

Comment (jamadden) [reply doc/index.rst:59]: I'm sorry, I wasn't clear. I was asking how one would pass options to the directive, like the examples just below of passing the `:ellipsis:` option. An example of that might be helpful, just to prove its possible.

Comment (ssbarnea) [reply doc/index.rst:59]: Now I got it! Done. Some options, much better.

**Comment by jamadden** [comment]
Thanks!

**Pull request merged**